src/app/trainees/trainee-list/trainee-list.component.html

```diff
@@ -10,9 +10,19 @@
     [matSortDirection]="sort.direction"
   >
     <ng-container *ngFor="let i of columnData" matColumnDef="{{ i.label }}">
-      <mat-header-cell *matHeaderCellDef [mat-sort-header]="i.name">{{
-        i.label
-      }}</mat-header-cell>
+      <ng-container *ngIf="i.enableSort; else loggedOut">
+        <mat-header-cell *matHeaderCellDef [mat-sort-header]="i.name">{{
+          i.label
+        }}</mat-header-cell>
+      </ng-container>
+      <ng-template #loggedOut>
```

Comment:
loggedOut? variable name seems out of context e.g ignoreSort, disableSort, sortNotRequired

Reply:
Just a thought

Reply:
ah good spot, cheers. pr updated